Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -334,7 +334,6 @@ public void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.EventFunction<
                 GameObject fallbackInput = fallbackInputStack.Peek();
                 if (ExecuteEvents.ExecuteHierarchy(fallbackInput, eventData, eventHandler) && eventData.used)
                 {
-                    return;
```

Comment:
Why was this removed?

Reply:
Good question.

Reply:
I'll re-add it in my incoming merge PR

Reply:
I guess it doesn't matter too much, since this is the end of the method either way. For consistency's sake, I'm re-adding it.

Reply:
Oh I see. It was the last return, which was redundant, and my formatting tool removed it.